versioned/tiered/tiered-impl/src/main/java/com/dremio/nessie/versioned/impl/condition/ExpressionFunction.java

```diff
@@ -107,7 +101,12 @@ public String asString() {
     if (name.binaryExpression) {
       return String.format("%s %s %s", arguments.get(0).asString(), name.protocolName, arguments.get(1).asString());
     }
-    return String.format("%s(%s)", name.protocolName, arguments.stream().map(v -> v.asString()).collect(Collectors.joining(", ")));
+    return String.format("%s(%s)", name.protocolName, arguments.stream().map(Value::asString).collect(Collectors.joining(", ")));
+  }
+
+  @Override
+  public ExpressionFunction alias(AliasCollector aliasCollector) {
```

Comment:
nit: moving a method just creates extra noise in a pr. please try to avoid in the future